app/src/main/java/de/westnordost/streetcomplete/quests/access_point_ref/AddAccessPointRef.kt

```diff
@@ -0,0 +1,42 @@
+package de.westnordost.streetcomplete.quests.access_point_ref
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddAccessPointRef : OsmFilterQuestType<AccessPointRefAnswer>() {
+
+    override val elementFilter = """
+        nodes with
+        (
+          (highway = emergency_access_point)
+          or
+          (emergency = access_point)
+        )
+        and !name and !ref and noref != yes and ref:signed != no and !~"ref:.*"
+    """
+
+    override val changesetComment = "Determine emergency access point refs"
+    override val wikiLink = "Tag:emergency=access_point"
```

Comment:
```suggestion
    override val wikiLink = "Key:ref"
```

Though https://github.com/streetcomplete/StreetComplete/blob/e67d1eb9107949162f5bc6a5b475631189a6d9ca/app/src/main/java/de/westnordost/streetcomplete/data/osm/edits/EditType.kt#L18 mentions that t can be also linking edited feature

Reply:
yes, this is fine